src/main/java/core/basesyntax/dao/FruitDaoImp.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.dao;
+
+import static core.basesyntax.db.Storage.storage;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public class FruitDaoImp implements FruitDao {
+    public Map<String, Integer> getStorage() {
+        return storage;
+    }
+
+    public void add(String type, Integer quantity) {
+        storage.put(type, quantity);
+    }
+
+    @Override
+    public void add(List<FruitTransaction> list) {
+        for (FruitTransaction item : list) {
+            storage.put(item.getFruit(), 0);
+        }
+    }
```

Comment:
Looks like these methods are not the same overridden method. They are completely different.